strawberryfields/apps/similarity.py

```diff
@@ -306,21 +335,132 @@ def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -
     return cardinality
 
 
+def _get_state(graph: nx.Graph, n_mean: float = 5, loss: float = 0.0) -> GaussianState:
+    r"""Embeds the input graph into a GBS device and returns the corresponding Gaussian state
+    """
+    modes = graph.order()
+    A = nx.to_numpy_array(graph)
+    mean_photon_per_mode = n_mean / float(modes)
+
+    p = sf.Program(modes)
+

```

Comment:
```suggestion

# pylint: disable=expression-not-assigned
```
The following lines are causing the codefactor/pylint problems: https://www.codefactor.io/repository/github/xanaduai/strawberryfields/pull/390. We're not doing anything wrong here, it's just the style used to define circuits in SF.

We can hence disable this pylint warning. This should only be done in special cases, and is done using
`# pylint: disable=error-to-disable`.